languages/python/oso/oso/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class AuthorizationError(Exception):
+    def __init__(self, actor, action, resource):
+        self.actor = actor
```

Comment:
I'm not seeing the benefit of providing actor/action/resource here. Maybe keep out for now and we can add back later?